README.md

```diff
@@ -223,6 +223,7 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
 | firebase                      | [jthegedus/asdf-firebase](https://github.com/jthegedus/asdf-firebase)                                           | [![Build](https://github.com/jthegedus/asdf-firebase/workflows/Build/badge.svg?branch=master)](https://github.com/jthegedus/asdf-firebase/actions)                                                                                |
 | fission                       | [virtualstaticvoid/asdf-fission](https://github.com/virtualstaticvoid/asdf-fission)                             | [![Build Status](https://github.com/virtualstaticvoid/asdf-fission/actions/workflows/workflow.yml/badge.svg?branch=main)](https://github.com/virtualstaticvoid/asdf-fission/actions/workflows/workflow.yml)                       |
 | flarectl                      | [flarectl](https://github.com/ORCID/asdf-flarectl)                                                              | [![Build](https://github.com/ORCID/asdf-flarectl/actions/workflows/build.yml/badge.svg)](https://github.com/ORCID/asdf-flarectl/actions/workflows/build.yml)                                                                      |
+| flatc                         | [flatc](https://github.com/TheOpenDictionary/asdf-flatc)                                                        | [![Build](https://github.com/ORCID/asdf-flarectl/actions/workflows/build.yml/badge.svg)](https://github.com/ORCID/asdf-flarectl/actions/workflows/build.yml)                                                                      |
```

Comment:
why are the badges coming from a different repository?

Reply:
Whoops! I copied this from another row and must have missed this. Just fixed!